src/SpaceStation.cpp

```diff
@@ -440,8 +446,50 @@ void SpaceStation::DockingUpdate(const double timeStep)
 	for (Uint32 i=0; i<m_shipDocking.size(); i++) {
 		shipDocking_t &dt = m_shipDocking[i];
 		if (!dt.ship) continue;
-		// docked stage is m_type->NumDockingPorts() + 1 => ship docked
-		if (dt.stage > m_type->NumDockingStages()) continue;
+		if (dt.stage > m_type->NumDockingStages()) {
```

Comment:
Yes: I did it as "extrastage" because I don't know how many stages there are. Orbitals encode each step between waypoints, then your ship are pos is interpolated; surface stations don't have any waypoint